CHANGELOG.md

```diff
@@ -15,17 +30,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
     - account for transitional states when handling state update conflicts
     - delete self under various error conditions indicative of a requeue
 
```

Comment:
Do we need a "Changed" for the SIGWINCH -> SIGUSR2 change? https://github.com/travis-ci/worker/commit/3ba889bb57e4953e5b5b08e96faec05fdefd90bf